packages/components/typography/src/SectionHeading/SectionHeading.tsx

```diff
@@ -43,14 +43,17 @@ function _SectionHeading<
       as={SECTION_HEADING_DEFAULT_TAG}
       testId={testId}
       marginBottom="spacingL"
-      fontWeight="fontWeightDemiBold"
-      fontColor="gray900"
+      fontWeight="fontWeightMedium"
+      fontColor="gray600"
       fontSize={density === 'high' ? 'fontSizeSHigh' : 'fontSizeS'}
       lineHeight={density === 'high' ? 'lineHeightSHigh' : 'lineHeightS'}
       className={cx(
         css({
-          letterSpacing: tokens.letterSpacingWide,
-          textTransform: 'uppercase',
+          letterSpacing: tokens.letterSpacingDefault,
+          textTransform: 'lowercase',
+          '&:first-letter': {
+            textTransform: 'uppercase',
+          },
```

Comment:
Do we need this at all?

Reply:
This is to make sure that the styling will be consistent, since because of the previous `uppercase` for everything, there may be places where the string is all lowercase or something like that